VideoLocker/res/values/styles.xml

```diff
@@ -23,9 +23,11 @@
         <item name="drawerArrowStyle">@style/DrawerArrowStyle</item>
         <item name="actionBarTabStyle">@style/TabsStyle</item>
         <item name="disableableButtonTextColor">@color/disableable_button_text</item>
+        <item name="android:listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
+        <item name="listChoiceBackgroundIndicator">@drawable/list_item_selector</item>
```

Comment:
The attribute with the `android` namespace is used in standard framework list views. However, the appcompat implementation introduced it's own attribute without namespace, which is used for backporting the material theme on pre-Lollipop devices, presumably for the sake of extending the support for pre-Honeycomb (where this attribute was introduced) without additional complexity.